terragrunt/aws/api/sentinel.tf

```diff
@@ -0,0 +1,17 @@
+locals {
+  api_log_group_arn            = "arn:aws:logs:${var.region}:${var.account_id}:log-group:/aws/lambda/${module.url_shortener_lambda.function_name}"
+  sentinel_forwarder_layer_arn = "arn:aws:lambda:ca-central-1:283582579564:layer:aws-sentinel-connector-layer:54"
```

Comment:
Is the account id the same for both staging and prod environments?

Reply:
Yup, for the layer ARN it is.  This is our SRE tools account where it gets published by this workflow step every time there's a change:
https://github.com/cds-snc/aws-sentinel-connector-layer/blob/b0e90a4dfafe0efafe305403478f60683bcdd9ea/.github/workflows/tf-apply.yml#L45-L48